distributed/tests/test_variable.py

```diff
@@ -39,6 +41,17 @@ def test_variable(c, s, a, b):
         assert time() < start + 5
 
 
+@gen_cluster(client=True)
+def test_delete_unset_variable(c, s, a, b):
```

Comment:
```suggestion
async def test_delete_unset_variable(c, s, a, b):
```

Reply:
The `yield` approach was common when we were also supporting Python 2.  It's still totally valid, but we've tended to default more towards using async/await.